lib/v2/screens/verification/components/create_passcode.dart

```diff
@@ -32,21 +28,20 @@ class _CreatePasscodeState extends State<CreatePasscode> {
       backgroundColor: AppColors.primary,
       shouldTriggerVerification: _verificationNotifier.stream,
       passwordEnteredCallback: (passcode) async {
-        BlocProvider.of<PasscodeBloc>(context).add(OnCreatePasscode(passcode: passcode));
+        BlocProvider.of<VerificationBloc>(context).add(OnCreatePasscode(passcode: passcode));
       },
       bottomWidget: Padding(
         padding: const EdgeInsets.only(top: 20),
         child: OutlinedButton(
-          style: ButtonStyle(
-            side: MaterialStateProperty.resolveWith<BorderSide>((states) => const BorderSide(color: AppColors.white)),
-            shape: MaterialStateProperty.resolveWith<OutlinedBorder>((_) {
-              return RoundedRectangleBorder(borderRadius: BorderRadius.circular(16.0));
-            }),
-          ),
-          child:
-              Text('Disable Pincode'.i18n, textAlign: TextAlign.center, style: Theme.of(context).textTheme.subtitle2),
-          onPressed: () => bloc.execute(DisablePasswordCmd()),
-        ),
+            style: ButtonStyle(
+              side: MaterialStateProperty.resolveWith<BorderSide>((states) => const BorderSide(color: AppColors.white)),
+              shape: MaterialStateProperty.resolveWith<OutlinedBorder>((_) {
+                return RoundedRectangleBorder(borderRadius: BorderRadius.circular(16.0));
+              }),
+            ),
+            child:
+                Text('Disable Pincode'.i18n, textAlign: TextAlign.center, style: Theme.of(context).textTheme.subtitle2),
```

Comment:
I don't really understand what the purpose of this button is, it is something that is among the things I need Nick to clarify for me.
Because it makes no sense to put `passcodeActive = false` and open the biometric again if the passcode is going to have first priority